{{cookiecutter.directory_name}}/apps/api/errors.py

```diff
@@ -79,37 +100,41 @@ def extra_headers(self) -> Tuple[Tuple]:
         return self._extra_headers
 
     @property
-    def payload(self) -> dict:
-        result = {
-            'title': self.title
-        }
-
-        if settings.DEBUG:
-            result['trace'] = traceback.format_exc().split('\n')
-
-        return result
-
-
-class ValidationException(ProblemDetailException):
-    def __init__(self, request, form: BaseForm):
-        super().__init__(request, _('Validation error!'), status=HTTPStatus.UNPROCESSABLE_ENTITY)
-        self._form = form
-
-    @property
-    def payload(self) -> dict:
-        payload = super(ValidationException, self).payload
-        payload['validation_errors'] = self._form.errors
-        return payload
+    def payload(self) -> ProblemDetail:
+        return ProblemDetail(
+            title=self.title,
+            type=self.type,
+            detail=self.detail,
+            trace=traceback.format_exc().split('\n') if settings.DEBUG else None
```

Comment:
Dajme asi prec, pretoze vzdy to iba zabudneme v DEBUG mode a potom nas maju za trotlov, ze vraciame tracebacks